relay/README.adoc

```diff
@@ -35,6 +35,69 @@ Then, invoke the start script by doing:
 And follow the prompts displayed in the console. After that, the relay client
 should be up and running.
 
+== Run using Docker
+
+Relay Maintianer can also be run from a Docker container.
+This instruction assumes the Bitcoin and Ethereum nodes are running and relay
+smart contract has been deployed to the Ethereum network.
+
+Create a directory where config files will be stored.
+
+Inside that directory create a TOML config file similar to the provided
+`config.toml.SAMPLE` file (let's assume its name is `config.toml`).
+
+The operator's keystore file (the "UTC" file) should also be put in that
+directory.
+
+Build a docker image and run it using the following commands:
+```
+docker build -t relay .
+docker run -v /path/to/config:/config -e OPERATOR_KEY_FILE_PASSWORD=password relay relay -config /config/config.toml start
+```
```

Comment:
Can we improve readability a little bit? My proposition:

Build the docker image using:
```
docker build -t relay .
```
Once done, run it using the following command:
```
docker run -v /path/to/config:/config -e OPERATOR_KEY_FILE_PASSWORD=password relay relay -config /config/config.toml start
```